src/server/game/Entities/GameObject/GameObjectData.h

```diff
@@ -575,14 +575,14 @@ struct GameObjectTemplate
     {
         switch (type)
         {
-        case GAMEOBJECT_TYPE_BUTTON: return button.large != 0;
-        case GAMEOBJECT_TYPE_QUESTGIVER: return questgiver.large != 0;
-        case GAMEOBJECT_TYPE_GENERIC: return _generic.large != 0;
-        case GAMEOBJECT_TYPE_TRAP: return trap.large != 0;
-        case GAMEOBJECT_TYPE_SPELL_FOCUS: return spellFocus.large != 0;
-        case GAMEOBJECT_TYPE_GOOBER: return goober.large != 0;
-        case GAMEOBJECT_TYPE_SPELLCASTER: return spellcaster.large != 0;
-        case GAMEOBJECT_TYPE_CAPTURE_POINT: return capturePoint.large != 0;
+        case GAMEOBJECT_TYPE_BUTTON:            return button.large != 0;
```

Comment:
tabs are partially still missing :D TC standard is using one additional tab before the case starts. See the other helpers in that struct